src/styles/theme.js

```diff
@@ -0,0 +1,21 @@
+// prettier-ignore
+const globalCss = {
+  'body': {
+    '@supports (height: 100svh)': { height: '100svh' },
+    '@supports not (height: 100svh)': { height: '100vh' },
+
+    '@media (max-width: 480px)': {
+      '@supports (width: 100svw)': { width: '100svw' },
+      '@supports not (width: 100svw)': { width: '100vw' },
+    },
+    '@media (min-width: 480px)': {
+      width: '480px',
+      margin: '0 auto',
+    },
+  },
+};
+
+const theme = {
+  globalCss,
+};
+export { theme };
```

Comment:
❗ theme.js는 디자인 토큰 객체를 저장해야 합니다.
```suggestion
const colors = {
  primary: '#3498db',
  gray: '#bdc3c7',
};

const fonts = {};

const theme = {
  colors,
  fonts,
};

export { theme };
```